README.md

```diff
@@ -90,47 +90,77 @@ Members have different degrees of love for each of the communities that they par
 
-> "Organizing a community meetup takes a lot of time and effort, so we give it an activity score of 8/10. Just attending a meetup is a 3/10, but publishing a summary counts for 2 more points."
+Since all activities default to a weight of 1, weighting activities is entirely optional. Some communities will require and welcome the complexity, while others will be just fine with the defauult weights. 
 
-Defining your community's list of activities and activity scores is one of the first things to do when starting with the Orbit Model. To get started, first make a table that looks like this:
+Make a table that looks like this:
 
-| Activity                        | Score (out of 10) |
+| Activity                        | Weight (out of 3) |
 | ------------------------------- | ----------------- |
-| Attended our conference         | 5                 |
-| Spoke at a meetup               | 7                 |
-| Opened a pull request on GitHub | 4                 |
-| Answered a forum question       | 2                 |
+| Attended our conference         | 1                 |
+| Spoke at a meetup               | 3                 |
+| Opened a pull request on GitHub | 1                 |
+| Had a pull request merged       | 3                 |
+| Answered a forum question       | 3                 |
 
-Then make another table that records the activities for each member. Capture the member's name, activity type, date, and activity score from the previous table.
+Defining your community's list of activities and activity weights is one of the first things to do when starting with the Orbit Model. 
 
-| Member Name       | Activity Type             | Date       | Score |
+Then, make another table that records the activities for each member. Capture the member's name, activity type, date, and activity weight from the previous table.
+
+| Member Name       | Activity Type             | Date       | Weight |
 | ----------------- | ------------------------- | ---------- | ----- |
-| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 5     |
-| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 4     |
-| Marie Monzo       | Answered a forum question | 2020-01-14 | 2     |
-| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 7     |
+| Jennifer Lewis    | Attended our conference   | 2019-11-14 | 1     |
+| Terrence Fletcher | Opened a pull request     | 2019-12-21 | 1     |
+| Marie Monzo       | Answered a forum question | 2020-01-14 | 3     |
+| Marie Monzo       | Had a pull request merged | 2020-01-20 | 3     |
+| Jennifer Lewis    | Spoke at a meetup         | 2020-02-11 | 3     |
+
+Once we have a member's activity history, we look at the weight of each activity along with how many months ago it ocurred to arrive at the value for each individual activity. 
+
+`value(month) =  (sum of activity values) * (0.9 ^ number of months ago)`
```

Comment:
We might want to explain the formula a bit, esp. the logarithmic scale. This could be a good time to introduce the concept of _decay_.

Reply:
+1 for introducing the concept and the rationale, that makes it more accessible to folks who don't have time to look at the formula.

Reply:
I'm adding a sub-section below the main flow. The goal is to unpack the idea but not disrupt the flow of the walk-thru. See what you think.

Reply:
@phacks @dzello Final thoughts on this?

Reply:
@patrickjwoods Getting it out in a separate section is a really good thing, as it helps simplify the reading. I would maybe add the updated formula at the end of the Actvity Decay section, e.g.:

> To take the activity decay into account in the love formula, you can multiply the resulting love by the decay: (then pasting the formula)